From: Debian iproute2 Maintainers <ah-iproute@debian.org>
Date: Mon, 10 Jun 2013 11:47:00 +0200
Subject: txtdocs

---
 doc/Makefile |    9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/doc/Makefile b/doc/Makefile
index b92957e..e1993b4 100644
--- a/doc/Makefile
+++ b/doc/Makefile
@@ -14,6 +14,7 @@ PAGESIZE=a4
 PAGESPERPAGE=2
 
 HTMLFILES=$(subst .sgml,.html,$(shell echo *.sgml))
+TXTFILES=$(subst .sgml,.txt,$(shell echo *.sgml))
 DVIFILES=$(subst .ps,.dvi,$(PSFILES))
 PDFFILES=$(subst .ps,.pdf,$(PSFILES))
 
@@ -28,6 +29,8 @@ dvi: $(DVIFILES)
 
 pdf: $(PDFFILES)
 
+txt: $(TXTFILES)
+
 print: $(PSFILES)
 	$(LPR) $(PSFILES)
 
@@ -65,9 +68,13 @@ print: $(PSFILES)
 %.html: %.sgml
 	$(SGML2HTML) $<
 
+%.txt: %.html
+	lynx -nolist -dump $< > $@
+		
+
 install:
 	install -m 0644 $(shell echo *.tex) $(DESTDIR)$(DOCDIR)
 	install -m 0644 $(shell echo *.sgml) $(DESTDIR)$(DOCDIR)
 
 clean:
-	rm -f *.aux *.log *.toc $(PSFILES) $(DVIFILES) *.html
+	rm -f *.aux *.log *.toc $(PSFILES) $(DVIFILES) *.html $(TXTFILES)
