From: Alexander Wirt <formorer@debian.org>
Subject: Add moo feature
Forwarded: not-needed

Index: quilt-oPPpWs/ip/ip.c
===================================================================
--- quilt-oPPpWs.orig/ip/ip.c	2011-11-23 21:53:54.000000000 +0100
+++ quilt-oPPpWs/ip/ip.c	2011-11-23 22:13:37.000000000 +0100
@@ -59,6 +59,20 @@
 	usage();
 }
 
+static int do_moo(int argc, char **argv)
+{
+	
+fprintf(stderr,
+"\n"
+" _ __ ___   ___   ___\n"
+"| '_ ` _ \\ / _ \\ / _ \\\n"
+"| | | | | | (_) | (_) |\n"
+"|_| |_| |_|\\___/ \\___/\n"
+"\n\n"
+"P.S. no real cows were harmed for this moo\n");
+	exit(1);
+}
+                       
 static const struct cmd {
 	const char *cmd;
 	int (*func)(int argc, char **argv);
@@ -83,6 +97,7 @@
 	{ "mrule",	do_multirule },
 	{ "netns",	do_netns },
 	{ "help",	do_help },
+	{ "moo",	do_moo }, 
 	{ 0 }
 };
 
