From: Alexander Wirt <formorer@debian.org>
Subject: Add moo feature
Forwarded: not-needed

Index: iproute/ip/ip.c
===================================================================
--- iproute.orig/ip/ip.c	2010-08-05 12:33:36.000000000 +0200
+++ iproute/ip/ip.c	2010-08-05 12:51:38.000000000 +0200
@@ -55,6 +55,20 @@
 	usage();
 }
 
+static int do_moo(int argc, char **argv)
+{
+	
+fprintf(stderr,
+"\n"
+" _ __ ___   ___   ___\n"
+"| '_ ` _ \\ / _ \\ / _ \\\n"
+"| | | | | | (_) | (_) |\n"
+"|_| |_| |_|\\___/ \\___/\n"
+"\n\n"
+"P.S. no real cows were harmed for this moo\n");
+	exit(1);
+}
+                       
 static const struct cmd {
 	const char *cmd;
 	int (*func)(int argc, char **argv);
@@ -78,6 +92,7 @@
 	{ "mroute",	do_multiroute },
 	{ "mrule",	do_multirule },
 	{ "help",	do_help },
+	{ "moo",	do_moo }, 
 	{ 0 }
 };
 
